idl/github.com/uber/cadence/admin.thrift

```diff
@@ -113,6 +125,22 @@ struct GetWorkflowExecutionRawHistoryResponse {
   40: optional i32 eventStoreVersion
 }
 
+struct GetWorkflowExecutionRawHistoryRequestV2 {
+  10: optional string domain
+  20: optional shared.WorkflowExecution execution
+  30: optional i64 (js.type = "Long") firstEventId
+  40: optional i64 (js.type = "Long") nextEventId
+  50: optional i64 (js.type = "Long") nextEventVersion
```

Comment:
this should be last event ID & last event version

Reply:
plz ignore the above comment.

plz also add some comments explaining what `next event` mean for the re-send use case

Reply:
or can you rename the var to
`startEventId`
`endEventId`
`endEventVersion`
and add a comment stating that this is inclusive & exclusive?

this is because next event ID means something different in history service

Reply:
Changed to use startEventId endEventId and endEventVersion